machine-learning-htc.shtml

```diff
@@ -0,0 +1,131 @@
+
+> This is a new "how-to" guide on the CHTC website. Recommendations and 
+> feedback are welcome via email (chtc@cs.wisc.edu) or by creating an 
+> issue on the CHTC website Github repository: [Create an issue](https://github.com/CHTC/chtc-website-source/issues/new)
+
+Overview
+========
+
+It's important to understand the needs of a machine learning job before submitting 
+it and have a plan for managing software. This guide covers: 
+
+1. [Considering job requirements](#1-job-requirements)
+2. [Recommendations for managing software](#2-software-options)
+
+# 1. Job Requirements
+
+Before digging into the nuts and bolts of software installation in the next section, 
+it's important to first consider a few other job requirements that might apply to 
+your machine learning job. 
+
+## A. Do you need GPUs?
+
+CHTC has about 4 publicly available GPUs and thousands of CPUs. When possible, using 
+CPUs will allow your jobs to start more quickly and to have many running at once. For 
+certain calculations, GPUs may provide a different advantage as some machine learning 
+algorithms are optimized to run significantly faster on GPUs. Consider whether you 
+would benefit from running one or two long-running calculations on a GPU or if your 
+work is better suited to running many jobs on CHTC's available CPUs. 
+
+If you need GPUs for your jobs, you can see a summary of available GPUs in CHTC and 
+how to access them here: 
+
+* [GPUs in CHTC](gpu-jobs.shtml)
+
+Note that you may need to use different versions of your software, depending on whether or 
+not you're using GPUs, as shown in the [software section of this guide](#2-software-options). 
+
+## B. How big is your data? 
+
+CHTC's usual data recommendations apply for machine learning jobs. If your job is using 
+an input data set larger than a few hundred MB or generating output files larger than 
+a few GB, you will likely need to use our large data 
+file share, Gluster. Contact the CHTC Research Computing Facilitators to get access and 
+read about Gluster here: 
+
+* [File Availability with Gluster](file-avail-gluster.shtml)
+
+## C. How long does your job run? 
+
+CHTC's default job length is 72 hours. If your task is long enough that you will 
+encounter this limit, contact the CHTC Research Computing Facilitators (chtc@cs.wisc.edu) 
+for potential work arounds. 
+
+## D. How many jobs do you want to submit? 
+
+Do you have the ability to break your work into many independent pieces? If so, 
+you can take advantage of CHTC's capability to run many independent jobs at once, 
+especially when each job is using a CPU. See our guide for running multiple jobs here: 
+
+* [Submitting Multiple Jobs Using HTCondor](multiple-jobs.shtml)
+
+# 2. Software Options
+
+Many of the tools used for machine learning, specifically deep learning and 
+convolutional neural networks, are sufficiently complex that our standard software 
+guides
```

Comment:
Some text is repeated here.  I read through quickly, and everything else here looks good enough to me for a first pass.

Reply:
oops, good catch - definitely started to edit a sentence there and didn't finish!